### PR TITLE
Mobile Quick User view card

### PR DESCRIPTION
**Pull request by JustSamuel**
Added a quick user over view (GEWIS m-number + name) display card which is shown on mobile. The idea is that people can show this at the bar to help with searching.

Missing translation-ish?
Also I believe a type error.

Closes#7

### CODE REVIEW

File: src/views/HomeView.vue
Comment (CodeNamedRobin) [inline src/views/HomeView.vue:1]: I smell a lint fix

File: src/components/UserInfoComponent.vue [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+});
+
+// Type guard
+function isGewisUser(user: UserResponse | GewisUserResponse): user is GewisUserResponse {
+  return (user as GewisUserResponse).gewisId !== undefined;
+}
+</script>
+
+<style scoped lang="scss">
+// Your styles here
```
Comment (CodeNamedRobin) [inline src/components/UserInfoComponent.vue:31]: I smell ChatGPT as well mhmm

**Review by CodeNamedRobin — APPROVED** [review]
LGTM!

**Comment by rinkp** [comment]
Closes #7

**Pull request merged**